README.md

```diff
@@ -74,151 +91,282 @@ public interface CrudRepository<T, ID> extends Repository<T, ID> {
-- `none`: Does not make any changes to the database.
+Now that we have added both those dependencies, if we were to run our application
+we would see an output similar to this:
 
-We will use the `create` value when we first create and persist `Teacher`
-instances and then switch to `none` for the read, update, and delete operations.
+![connect-to-database](https://curriculum-content.s3.amazonaws.com/spring-mod-1/repository/connect-to-postgresql-sport-db.png)
 
-## Create Entity and Repository
+The lines in this log we want to look at have the word "hibernate" in it. These
+show us that the application is connecting to the database. If there were any
+issues with connecting to the database, we might see a stack trace of errors.
 
-Create a `models` and `repositories` package in the
-`com.example.springdatajpaexample` package (or whatever the main package is in
-your project).
+## Create the Entity Class
 
-Next, create a `Teacher` class in the `models` package and add the following
-code:
+In the `entity` package, go ahead and create a `FootballTeam` class with the
+following code:
 
 ```java
-package com.example.springdatajpaexample.models;
+package com.example.springdatademo.entity;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
 
 import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
+import javax.persistence.Table;
 import javax.persistence.Id;
+import javax.persistence.GeneratedValue;
 
 @Entity
-public class Teacher {
+@Getter
+@Setter
+@NoArgsConstructor
+@Table(name = "football_team")
+public class FootballTeam {
+
     @Id
     @GeneratedValue
     private int id;
 
-    private String name;
+    private String teamName;
 
-    public String getName() {
-        return name;
-    }
+    private int wins;
 
-    public void setName(String name) {
-        this.name = name;
-    }
+    private int losses;
 
-    @Override
-    public String toString() {
-        return "Teacher{" +
-                "id=" + id +
-                ", name='" + name + '\'' +
-                '}';
-    }
+    private boolean currentSuperBowlChampion;
 }
 ```
 
-Now that we have the entity, we can create the repository. Create a class called
-`TeacherRepository` in the `repositories` package and add the following code:
+## Create the Repository Class
+
+Now that we have the entity, we can create the repository. Create an interface
+called `FootballRepository` in the `repository` package and add the following
+code:
 
 ```java
-package com.example.springdatajpaexample.repositories;
+package com.example.springdatademo.repository;
 
-import com.example.springdatajpaexample.models.Teacher;
+import com.example.springdatademo.entity.FootballTeam;
 import org.springframework.data.repository.CrudRepository;
 
-public interface TeacherRepository extends CrudRepository<Teacher, Integer> {
+public interface FootballRepository extends CrudRepository<FootballTeam, Integer> {
```

Comment:
Does this need the @Repository annotation?

Reply:
Yes it does. Thanks for catching this!